backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -37,8 +37,8 @@ export const listMessages = async (
       name: recipient_name,
       mobile: recipient,
       data: JSON.stringify(remainingParams),
-      passcode: row.govsgVerification?.passcode ?? '****',
-      sent: row.sendAttemptedAt,
+      passcode: row.govsgVerification?.passcode ?? '',
```

Comment:
Curious, why did we decide to handle this in the FE instead?

Reply:
Not sure if I am answering your qn:

Because I prefer to check `!value` than `value === 'Not created yet'`

Lmk if I'm not!

Reply:
My understanding of this code change is that previously we were resolving the display value entirely in this file (passcode = `****` if `row.govsgVerification?.passcode` is null) and then simply displaying the value of `passcode` on the FE. 

But now, we make it such that `passcode` could be an empty string and we check for this value in the FE instead (in `GovsgMessages.tsx`) before deciding what to display to the user.

Reply:
Yup, your understanding is correct.

And now we want to show `Not created yet` instead of `****`. I rather do an empty string check.

Previously, I made the backend return `****` because I was simply lazy to do an empty string check. But now, it's unavoidable.

Reply:
Alright, I understand now. Thank you!